test/bin/scenario.sh

```diff
@@ -33,6 +33,7 @@ VNC_CONSOLE=${VNC_CONSOLE:-false}  # may be overridden in global settings file
 TEST_RANDOMIZATION="all"  # may be overridden in scenario file
 TEST_EXECUTION_TIMEOUT="30m" # may be overriden in scenario file
 SUBSCRIPTION_MANAGER_PLUGIN="${SUBSCRIPTION_MANAGER_PLUGIN:-${SCRIPTDIR}/subscription_manager_register.sh}"  # may be overridden in global settings file
+RUN_HOST_OVERRIDE=""  #Target any given VM for running scenarios
```

Comment:
Nit: consistency with the comment above
```suggestion
RUN_HOST_OVERRIDE=""  # target any given VM for running scenarios
```

Reply:
Done !!